WcaOnRails/db/migrate/20211126105123_update_preferred_formats_again.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class UpdatePreferredFormatsAgain < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
Does `change` actually works here?
It's meant to be a way to provided reversible actions so that we can use `db:migrate` or `db:rollback` (eg: `create_table` would automatically be transformed into a drop table when doing a rollback).
I realize previous similar migrations have this though...
One way of doing this "properly" is to provide an `up` and `down` methods which are used when migrating/rollback-ing.

Reply:
I think it does actually work, as I've done a rollback and remigrated and it was working as expected